modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/configuration/schema/PageMemoryDataRegionConfigurationSchema.java

```diff
@@ -57,10 +56,6 @@ public class PageMemoryDataRegionConfigurationSchema {
     @InjectedName
     public String name;
 
-    @Immutable
-    @Value(hasDefault = true)
-    public int pageSize = 16 * 1024;
-
     @Value(hasDefault = true)
     public boolean persistent = false;
```

Comment:
I believe it would be nice to make this configuration polymorphic. Volatile regions don't need "delay...Write" config, persistent regions don't need minimal size/eviction, and so on

Reply:
Tried to make a polymorphic configuration.